sdk/storage/azure-storage-file/src/test/java/com/azure/storage/file/spock/DirectoryAPITests.groovy

```diff
@@ -21,6 +25,8 @@ class DirectoryAPITests extends APISpec {
     def directoryPath
     def shareName
     static def testMetadata
+    static def smbProperties
+    static def filePermission = "O:S-1-5-21-2127521184-1604012920-1887927527-21560751G:S-1-5-21-2127521184-1604012920-1887927527-513D:AI(A;;FA;;;SY)(A;;FA;;;BA)(A;;0x1200a9;;;S-1-5-21-397955417-626881126-188441444-3053964)S:NO_ACCESS_CONTROL"
```

Comment:
Is any of this file permission string sensitive information or contain any secrets? If so we'll need to find another way to get this information for running the tests (also you'll need to regenerate any secrets this used since they're publicly visible).

Reply:
This isn't sensitive information from what I know of.